web/src/message_list.js

```diff
@@ -373,20 +373,28 @@ export class MessageList {
     // Maintains a trailing bookend element explaining any changes in
     // your subscribed/unsubscribed status at the bottom of the
     // message list.
-    update_trailing_bookend() {
+    update_trailing_bookend(force_render = false) {
         this.view.clear_trailing_bookend();
         if (this.is_combined_feed_view) {
             return;
         }
+
         const stream_name = narrow_state.stream_name();
         if (stream_name === undefined) {
+            // For unknown stream narrow, we don't show trailing bookend.
```

Comment:
This isn't just the unknown stream case, it's also if you're just looking at a non-stream view.